bcda/servicemux/servicemux_test.go

```diff
@@ -82,7 +82,7 @@ var testHandler http.HandlerFunc = func(w http.ResponseWriter, r *http.Request)
 	}
 }
 
-func (s *ServiceMuxTestSuite) TestServe_serveHTTPS() {
+func (s *ServiceMuxTestSuite) TestServeServeHTTPS() {
```

Comment:
Can we change the 'ServeServe' part of these names to just 'Serve'. When they were separated with _ , it was following a given/when/should pattern in test naming. Without the _, it's hard to read.

Reply:
done!